src/issuance.rs

```diff
@@ -669,6 +735,16 @@ mod tests {
         assert!(!supply.is_finalized);
     }
 
+    #[test]
+    fn test_verify_supply_invalid_for_asset_base_as_identity() {
```

Comment:
remove `test_` here and all the surrounding tests.

Reply:
Removed.